cli/azd/pkg/project/framework_service_docker.go

```diff
@@ -214,7 +214,7 @@ func getDockerOptionsWithDefaults(options DockerProjectOptions) DockerProjectOpt
 	}
 
 	if options.Platform == "" {
-		options.Platform = "amd64"
+		options.Platform = "linux/amd64"
```

Comment:
looks like `linux/amd64` should be a constant and be consumed from every where from pkg/project

Reply:
What if we refactored this to `project.Load` to set the default config value? That seems like the right thing to do. Can be a follow-up. With that in mind, I'd worry less about the constant.

Reply:
Let's hold off with this for the time being.  I have doubts that any framework/service target related defaults should be handled directly at the project root since this is more of a concern/implementation detail of the consuming famework/service target.